swagger_server/jobs/pull_topo_changes.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import json
+import logging
+import os.path
+import sys
+import threading
+import time
+import urllib.request
+
+sys.path.append(
+    os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir))
+)
+
+from messaging.rpc_queue_producer import *
+from utils.db_utils import *
+
+DOMAIN_CONTROLLER_PULL_URL = os.environ.get("DOMAIN_CONTROLLER_PULL_URL")
+DOMAIN_CONTROLLER_PULL_INTERVAL = os.environ.get("DOMAIN_CONTROLLER_PULL_INTERVAL")
+logger = logging.getLogger(__name__)
+
+
+def main():
+    db_instance = DbUtils()
+    db_instance.initialize_db()
+
+    process_domain_controller_topo(db_instance)
+
+
+def process_domain_controller_topo(db_instance):
+    while True:
+        latest_topology_exists = False
```

Comment:
Can't we do `if latest_topology is None:...` instead of using this boolean?

Reply:
This is just to check the status later [here](https://github.com/atlanticwave-sdx/sdx-lc/blob/41a4696c3322128da5d5e84805558c951d759cfb/swagger_server/jobs/pull_topo_changes.py#L66), to sleep if new topology has same version as the new topology. I feel this is a bit easier to read.